client/src/legacy/TinyMCE_sslink-internal.js

```diff
@@ -0,0 +1,128 @@
+/* global tinymce, ss */
+import i18n from 'i18n';
+import TinyMCEActionRegistrar from 'lib/TinyMCEActionRegistrar';
+import React from 'react';
+import ReactDOM from 'react-dom';
+import { ApolloProvider } from 'react-apollo';
+import jQuery from 'jquery';
+import ShortcodeSerialiser from 'lib/ShortcodeSerialiser';
+import { createInsertLinkModal } from 'containers/InsertLinkModal/InsertLinkModal';
+import { provideInjector } from 'lib/Injector';
+
+// Link to external url
+TinyMCEActionRegistrar.addAction('sslink', {
+  text: i18n._t('CMS.LINKLABEL_PAGE', 'Link to page'),
+  // eslint-disable-next-line no-console
+  onclick: (editor) => editor.execCommand('sslinkinternal'),
+});
+
+const plugin = {
+  init(editor) {
+    editor.addCommand('sslinkinternal', () => {
+      const field = window.jQuery(`#${editor.id}`).entwine('ss');
+
+      field.openLinkInternalDialog();
+    });
+  },
+};
+
+const modalId = 'insert-link__dialog-wrapper--internal';
+const sectionConfigKey = 'SilverStripe\\CMS\\Controllers\\CMSPageEditController';
+const formName = 'editorInternalLink';
+const InsertLinkInternalModal = provideInjector(createInsertLinkModal(sectionConfigKey, formName));
+
+jQuery.entwine('ss', ($) => {
```

Comment:
From what I can see the use of entwine here is to add `openLinkInternalDialog()` to the DOM and `openLinkInternalDialog()` is only being used in at line 24 in the same file so instead of attached a function to the DOM using entwine, it can just be a normal function.